ui/tests/unit/adapters/job-test.js

```diff
@@ -383,6 +383,10 @@ module('Unit | Adapter | Job', function(hooks) {
     const region = 'region-2';
     window.localStorage.nomadActiveRegion = region;
 
+    // Regions are fetched in the before hook, so manually dirty the activeRegion
+    // instead of repeating what is in the beforeEach here.
+    this.system.notifyPropertyChange('activeRegion');
```

Comment:
This jumped out at me a bit, and I had to go and read the system service to understand what's going on here. Is there any alternative to `notifyPropertyChange` here? For example, might it make sense to call `this.system.set('activeregion', region)`, or are you trying to test that the computed property fetches the region from local storage?

Reply:
Specifically trying to test that the computed property fetches the region from local storage. 

The alternative is to get namespaces after setting the localStorage property, but that means all my URL checks need to include namespaces, or omit the first entry.

This is not my favorite but it made for the smallest diff and least repetitive assertions. Maybe it's still worth refactoring.